app/src/main/java/com/dkanada/gramophone/model/DirectPlayCodec.java

```diff
@@ -16,9 +16,11 @@ public DirectPlayCodec(String codecName, String title, String value, boolean sel
     public enum Codec {
         FLAC("FLAC","flac|flac"),
         MP3("MP3", "mp3|mp3"),
-        AAC("AAC", "m4a|aac"),
-        OGG("OGG", "ogg|vorbis"),
-        MKA("MKA", "mka|opus");
+        OPUS("Opus", "opus|opus"),
+        AAC("M4A-AAC", "m4a|aac"),
+        OGG("OGG-Vorbis", "ogg|vorbis"),
+        OOPUS("OGG-Opus", "ogg|opus"),
+        MKA("MKA-Opus", "mka|opus");
```

Comment:
How familiar are you with Android development? Before this is merged, I'd like to modify this enum so that the container and codec titles are separated, and the codec names show up as "subtitles" in grey next to the container names. If you're not used to Android I can probably add that at some point.

Reply:
I'm unfortunately not very experienced in android development